src/components/comment/commentCard.js

```diff
@@ -0,0 +1,20 @@
+const CommentCard = (props) => {
+
+  const { author, body, vote, created_at } = props;
+
+  return (
+    <div className="comment_card">
```

Comment:
These cards should be `li`'s as there's semantic meaning to them rather than just `div`'s.